### ISSUE TITLE
"Few" Suggestions

### ISSUE THREAD

**Issue by Remyrios** [imrgr/rise-of-the-pack#11]
## Mod Suggestions:
- [Integration Foregoing](https://minecraft.curseforge.com/projects/integration-foregoing)
- [Forestry](https://minecraft.curseforge.com/projects/forestry)
- [Binnie's Mods](https://minecraft.curseforge.com/projects/binnies-mods)
- [Gendustry](https://minecraft.curseforge.com/projects/gendustry)
- [Magic Bees](https://minecraft.curseforge.com/projects/magic-bees)
- [Exchangers](https://minecraft.curseforge.com/projects/exchangers)
- [BetterFps](https://minecraft.curseforge.com/projects/betterfps)
- [RFTools Control](https://minecraft.curseforge.com/projects/rftools-control)
- [XNet](https://minecraft.curseforge.com/projects/xnet)
- [FoamFix](https://asie.pl/files/mods/FoamFix/)
- [Unlimited Chisel Works](https://minecraft.curseforge.com/projects/unlimited-chisel-works)
- [FPS Reducer](https://minecraft.curseforge.com/projects/fps-reducer)
- [Psi](https://minecraft.curseforge.com/projects/psi)
- [Morph-o-Tool](https://minecraft.curseforge.com/projects/morph-o-tool)
- [Akashic Tome](https://minecraft.curseforge.com/projects/akashic-tome)
- [NotEnoughIDs](https://minecraft.curseforge.com/projects/notenoughids)
- [Tomb Many Graves 2](https://minecraft.curseforge.com/projects/tomb-many-graves-2)
- [Wawla](https://minecraft.curseforge.com/projects/wawla-what-are-we-looking-at)
- [Wireless Crafting Terminal](https://minecraft.curseforge.com/projects/wireless-crafting-terminal)
- [JAOPCA](https://minecraft.curseforge.com/projects/jaopca)
- [Spartan Shields](https://minecraft.curseforge.com/projects/spartan-shields)
- [Astral Sorcery](https://minecraft.curseforge.com/projects/astral-sorcery)
- [Ender Storage](https://minecraft.curseforge.com/projects/ender-storage-1-8)
- Ender Tanks (Removal)

## Script Changes
- Making Transmutation Table recipe harder or removal of ProjectE and adding recipes for creative items/blocks

*New recipe for Transmutation Table - Chaotic Tier DE Infusion - Center: Energy Condenser MK2; Injectors: 2x Ultimate Catalyst (Glowing ones), Ultimate Crafting Table, 4x Creative Storage Upgrades, Snow Queen Trophy, 2x /dank/null/MK VI (Or 1 /dank/null and Mantle of Stars with Horologium Constelation if Astral Sorecry added), ME 16384k Storage Cell, Ultimate Induction Provider*

*Changes in Energy Condenser MK2 Recipe - Replacing Wyvern Cores with Chaoti Cores; Replacing Pixie Dusts to Gaia Spirit Ingots; Replacing Terrasteel to Enderium, Signalum, Lumium and Steel Singularities; Replacing 128³ Spatial Storage Cell with Draconic Flux Capacitor*

**Comment by Watschman** [comment]
My thoughts on the suggestion:

Mod Changes
----------------------------
- Integration Foregoing
   - Sure more Modcompatibility is always nice
- Forestry, Binnie's Mods, Gendustry, Magic Bees
   - A lot of people dislike bees, I personally like them but it would require a lot of retrogen to already created worlds, it might also break certain progressions due to the fact that you can obtain almost everything from bees.
- Exchangers
   - Might be a tech alternative to Botanias Rod of the Shifting Crust, but I don't necessarily see the need to add it since you need Botania to do Tech progression anyway.
- BetterFPS
   - I never had FPS issues, only CPU, therefore I moved my world to my server. I think when people need that they can just include it on their own, similar to Optifine. If they want it they can install it afterwards, so no need to include it in the pack.
- RFTools Control / Psi
   - It looks interesting enough, though I am not sure how powerful it is and if it breaks progression of any sort due to the fact that programming based mods usually can be very powerful if used by the right person.
- XNet
   - Seems to be very powerful judging from the description, instant item, energy, fluid transport would be very very late game, otherwise any other pipes or ducts become useless, therefore very progression breaking.
- Foam Fix
   - I was unable to retrieve much information regarding the mod but judging from the Source Code description it is an memory usage optimization Mod. My opinion would be similar to the BetterFPS Mod, If those Mods are done right people should be able to connect with them installed to servers which have them not installed.
- Unlimited Chisel Works
   - Seems like it is only adding cosmetic blocks, therefore sure why not
- FPS Reducer
   - The same as BetterFPS and Foam Fix, but here I totally dislike it, I would remove it on my client if it were added to the pack
- Morph-o-tool
   - Seems awesome since apparently in 1.8+ everyone dropped cross-wrench compatibility for some reason, therefore I don't need 4 different wrenches variations in my Tool Backpack
- Akashic Tome
   - Similar to Morph-o-tool, I have no reason to oppose the suggestion
- Not Enough ID's
   - I am not sure why that would be needed. Not quite sure if the Modpack will hit 16bit of Items or Blocks. Unless I miss any other usage of the mod besides of possibly making worlds incompatible
- Tomb Many Graves 2
   - Redundant, the pack already has a working grave mod.
- Wawla 
   - More information, why not when it is compatible, which it should be.
- Wireless Crafting Terminal
   - Destroys the intention of Applied Energistics of doing automated crafting, when you want wireless access to your items, therefore I am absolutely not a fan.
- JAOPC
   - Seems like a port attempt of AOBD, not sure how good it is but why not, compatibility is always nice to have
- Spartan Shields
   - I don't really care about that one.
   - PS: who uses shields anyway?
- Astral Sorcery
   - I think that mod was inspired by Thaumcraft, I liked Thaumcraft but adding it to the pack would mean a complete revamp of the progression and a lot of retrogen, if that is even possible without breaking worlds.
- Ender Storage
   - We already have Multi-Storage and Ender Tanks. Ender Storage would combine does Mods into one, if it has still the same functionality as in 1.7.
   - So I would sign that suggestion

Script Changes
-----------------------------------------
As for the recipe changes:
I don't really care much but I originally intended to not spend more than 100-200hours into the pack, so I will definitely not reach those points if it gets even harder.

TL;DR:
---------------------------------------------
Some of these suggestions are pretty good while others seem to be either redundant or unnecessary to me.
But those are just my opinions on it, it would have been nice if there was a comment to every change, why it should have been added/removed/altered to the pack and what benefits it would bring in the original post.

**Comment by Remyrios** [comment]
- **Forestry and etc.**
I suggested this because Forestry isn't just bees. Of course bees're cool and these could be available in the mid/late game.
- **XNet**
Helps to automate a lot, it allows for a lot of logic systems, but yes it's kinda a bit owerpowred with unlimited trasnfer you could add it to the late-game
- **FoamFix, BetterFPS**
Even if you have a good fps without them they won't break anything and having these kinds of mods in modpack is always nice
- **RFTools Control**
It's similar to the XNet but a bit less powerful
- **Exchangers**
 It's a bit more tiered and tbh rod from Botania should require one of the basic exchangers not exchangers require rod from Botania
- **FPS Reducer**
Why would you remmove it? Like it *speeds up your computer* when you have minecraft in background
- **Psi**
Magic mod in which you program your own spells and it isn't that overpowered.
- **NotEnoughIDs**
I suggested this because if you add Unlimited Chisel Works you probably will run out of ids
- **Tomb Many Graves**
I haven't seen GraveStone mod, my bad
- **Wireless Crafting Terminal**
Maybe yes? But sometimes it's a bit annoying when you need something small you're 1k blocks away and you need to teleport to your base or even walk to it
- **Spartan Shields**
I use shields lol. They are very useful at early/mid game
- **Astral Sorcery**
It needs retrogen, it's very different from Thaumcraft, but you could add it in the big update not a small like one
- **Ender Storage**
For me it's better than the MultiStorage and EnderTanks. The only thing that i like from MultiSotrage are these stone containers (Don't know their name)
- **Script Changes**
ProjectE is still to easy it should be removed or changed recipes as i said. Maybe not to these which i suggested but it should be at least 2x harder to make.

**P.S. i will describe mods in original post later if i have time**
